eox_essence/wrapper/edxapp/enrollments.py

```diff
@@ -0,0 +1,11 @@
+"""Wrapper file for site_configuration open-edx djangoapp.
+
+This file contains all the wrapped modules for the site_configuration djangoapp.
+https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/enrollments
+
+Attributes:
+    api: Open Edx Enrollment api module.
+"""
+from eox_essence.wrapper.edxapp import get_module
+
+api = get_module('enrollments', 'api')
```

Comment:
Right now, this implementation is the absolute simplest it can be. Since it only maps the enrollments module and return it.

How do you imagine this if the implementation of the enrollments.api module changes in a way that is no longer simple to maintain?

Would this file have to change, or would the interface change in that moment? If so, the same is valid there. How should we implement the interface once there is more than 1 implementation